src/_locales/en-US.json

```diff
@@ -63,6 +63,8 @@
   "playback-label-volume-down": "Decrease Volume",
   "playback-label-volume-up": "Increase Volume",
 
+  "settings-option-invert-tray-icon": "Invert Tray Icon Color",
```

Comment:
In order to make the tests pass you have to add this key to all the other JSON files, just leave the value empty.